app/dashboards/feedback_dashboard.rb

```diff
@@ -0,0 +1,52 @@
+require "administrate/base_dashboard"
+
+class FeedbackDashboard < Administrate::BaseDashboard
+  def show_in_sidebar?
+    true
+  end
+
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    email: Field::String,
+    url: Field::String,
+    comment: Field::Text,
+    created_at: Field::DateTime
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = [
+    :id,
+    :email,
+    :url,
+    :created_at
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = [
+    :id,
+    :email,
+    :role,
+    :organisation,
```

Comment:
You added role and organisation here but they should also be added in `ATTRIBUTE_TYPES`.